src/views/htmlcontent/src/html/sqlOutput.ejs

```diff
@@ -19,14 +19,14 @@
         font-size: var(--font-size);
       }
     </style>
+    <link rel="stylesheet" type="text/css" href="lib/css/slick.grid.css"/>
+    <link rel="stylesheet" type="text/css" href="lib/css/SlickGrid.css"/>
```

Comment:
have you verified there are not other style changes that result from reordering these CSS files?